src/daal4py.cpp

```diff
@@ -427,24 +491,22 @@ daal::data_management::NumericTablePtr make_nt(PyObject * obj)
                 if(PyErr_Occurred()) {PyErr_Print(); throw std::runtime_error("Python Error");}
 
                 if(np_indcs && np_roffs && np_vals && nr && nc) {
-                    // for now, increment indcs by 1
                     size_t * c_indcs = (size_t*)array_data(np_indcs);
                     size_t n = array_size(np_indcs, 0);
-                    for(size_t i=0; i<n; ++i) c_indcs[i] += 1;
-
+                    size_t * c_indcs_one_based = (size_t*)malloc(n * sizeof(size_t));
```

Comment:
Use service functions from DAAL: https://github.com/oneapi-src/oneDAL/blob/master/cpp/daal/include/services/daal_memory.h

```suggestion
                    size_t * c_indcs_one_based = static_cast<size_t*>(daal::services::daal_malloc(n * sizeof(size_t));
```